models/sources/snapshots.sql

```diff
@@ -14,6 +14,7 @@ select
     cast(null as {{ type_string() }}) as package_name,
     cast(null as {{ type_string() }}) as path,
     cast(null as {{ type_string() }}) as checksum,
-    cast(null as {{ type_string() }}) as strategy
+    cast(null as {{ type_string() }}) as strategy,
+    cast(null as {{ type_json() }}) as meta
```

Comment:
@NiallRees this is some kind of a bug, there is no trailing whitespace in these two files. SQLFluff lints them OK for me locally. Could you pls take a look?